test/arbitrum/compounders/curve/CurveCompounderBaseArbitrumTest.sol

```diff
@@ -96,6 +96,34 @@ contract CurveCompounderBaseArbitrumTest is Test, AddressesArbi {
         _testRedeemSingleUnwrappedETH(_sharesAlice, _sharesBob, _sharesCharlie);
     }
 
+    function _testDepositCap(address _asset, uint256 _amount) internal {
+        
+        // ------------ Get _asset ------------
+        
+        uint256 _underlyingAlice = _getAssetFromETH(alice, _asset, _amount);
+        uint256 _underlyingBob = _getAssetFromETH(bob, _asset, _amount);
+        uint256 _underlyingCharlie = _getAssetFromETH(charlie, _asset, _amount);
+
+        // ------------ Deposit ------------
+
+        _testDepositSingleUnwrapped(_asset, _underlyingAlice, _underlyingBob, _underlyingCharlie);
+
+        // ------------ Harvest ------------
+        
+        // Fast forward 1 month
+        skip(216000);
```

Comment:
I guess we do not have to skip for testing the cap. Just saying.

Reply:
The `skip` cheatcode skips over blocks (see comment on L113 - skips about a month forward) - we do that to simulate the effects of accumulating rewards.

Reply:
My thinking was that this particular case was to test the deposit cap and in theory no rewards have to be accrued in order to test that functionality.